chainercv/links/utils/bbox/bbox_overlap.py

```diff
@@ -0,0 +1,42 @@
+from chainer import cuda
+
+
+def bbox_overlap(bbox, query_bbox):
+    """Calculate overlap coefficients between bounding boxes.
+
+    The coefficient is calculated as a ratio of area of the intersection
+    and area of the union.
+
+    This function accepts both :obj:`numpy.ndarray` and :obj:`cupy.ndarray` as
+    inputs.
```

Comment:
Please note that both of two arrays must be same type. For example, this function will not work when `bbox` is `numpy.ndarray` and `query_bbox` is `cupy.ndarray`.